easybuild/easyconfigs/a/ASAGI/ASAGI-1.0-foss-2023a.eb

```diff
@@ -0,0 +1,63 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jonney Huang <jonney.huang@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'CMakeMake'
+
+name = 'ASAGI'
+local_commit_id = 'f672507'
+version = '1.0'
+
+homepage = 'https://github.com/TUM-I5/ASAGI'
+description = """ a pArallel Server for Adaptive GeoInformation.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/TUM-I5/',
+        'repo_name': 'ASAGI',
+        'commit': local_commit_id,
+        'recursive': True,
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+patches = [
+    ('ASAGI-CMakeList.patch', 0),
+    ('ASAGI-level.h.patch', 0),
+]
+checksums = [
+    {'ASAGI-1.0.tar.gz': '703927701a04850a6da9754427dcf429e455ca825673587f5df389980d7452d7'},
```

Comment:
Checksums for sources downloaded via `git_config` were unstable with EasyBuild 4.x, but with EasyBuild 5.x they should be reproducible, so this should be updated.

That said, I'm actually not getting a reproducible checksum for this tarball across systems, it seems like that's because of the `.git/modules/submodules` that's included in the tarballs...

@lexming Is that a bug in framework, or is it simply not possible to have reproducible checksums when `recursive` option is enabled in `git_config`?

Reply:
```suggestion
    {'ASAGI-1.0.tar.gz': None},
```